src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -1211,4 +1244,14 @@ private boolean needNumFilteredGenotypes(){
                 maxFractionFilteredGenotypes != MAX_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE ||
                 minFractionFilteredGenotypes != MIN_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
     }
+
+    /**
+     * Need the number of no-call genotyped samples?
+     *
+     * @return true if any of the filtered genotype samples arguments is used (not the default value), false otherwise
+     */
+    private boolean needNoCallGenotypes(){
+        return maxNOCALLnumber != MAX_NOCALL_NUMBER_DEFAULT_VALUE ||
```

Comment:
I would prefer a null value (with an error message that tells the user to specify a number if they didn't).